shared/general/labels.md

```diff
@@ -1,13 +1,28 @@
+__Note:__ If you have devices in your app that have a supervisor version lower than 7.22.0, then you should use the `io.resin.features.` form of the labels to ensure that all devices obey the label. Earlier supervisor version will not understand the `io.balena.features` label.
```

Comment:
microscopic nit:
```suggestion
__Note:__ If you have devices in your app that have a supervisor version lower than 7.22.0, then you should use the `io.resin.features.` form of the labels to ensure that all devices obey the label. Earlier supervisor versions will not understand the `io.balena.features` label.
```

Reply:
fixed in cb44b287684d0fed3483305b860cf31e2b55f015